src/components/dashboard/Dashboard.jsx

```diff
@@ -55,9 +53,7 @@ const Dashboard = (props) => {
           <h2>Recent templates</h2>
           <ResourceTemplateSearchResult
             results={historicalTemplates}
-            handleClick={handleNew}
-            handleEdit={handleEdit}
-            handleCopy={handleCopy}
+            errorKey={dashboardErrorKey}
```

Comment:
Notice that we're passing around `errorKey` rather than the various useResource functions.